File: JavaScript-Problems/ProblemSolvingLevel1/problem2/problem2.js
function printName(){
    const nameInput = document.getElementById('nameInput');

    const name = nameInput.value.trim();

    const outputArea = document.getElementById('outputArea');

    if(name){
        outputArea.innerHTML = `<span class="font-bold text-lg">Name:</span> ${name}`;
        nameInput.value = '';
    }
    else{
        outputArea.innerHTML = `<span class="text-red-500">Please enter a valid name before submitting.</span>`;
    }

    document.getElementById('nameInput').addEventListener('keypress', function(event){
        if (event.key === 'Enter'){
            event.preventDefault();
            document.getElementById('printButton').click();
        }
    });
}

File: C#-Problems/ProblemSolvingLevel1/problem2.cs
using System;

class Problem2
{
    static void PrintName()
    {
        string? Name;
        Console.WriteLine("Please Enter Your Name:");

        // Name Enter From User 
        Name = Console.ReadLine();

        Console.WriteLine("Your Name Is: " + Name);
    }

    static void Main(string[] args)
    {
        // Calling The Function 
        PrintName();
    }
}